src/components/dialog/content/TopUpCreditsDialogContentWorkspace.vue

```
@@ -0,0 +1,313 @@
+          class="flex items-center gap-1 text-sm text-muted-foreground no-underline transition-colors hover:text-base-foreground"
+        >
+          {{ $t('credits.topUp.viewPricing') }}
+          <i class="icon-[lucide--external-link] size-4" />
+        </a>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { useToast } from 'primevue/usetoast'
```

ESLint

[error] Unable to resolve path to module 'primevue/usetoast'.

(import-x/no-unresolved)